Makefile

```diff
@@ -246,6 +246,13 @@ verify-crd:
 	$(diff) '$(tmp_file)' '$(CRD_PATH)' || (echo 'File $(CRD_PATH) is not up-to date. Please run `make update-crd` to update it.' && false)
 .PHONY: verify-crd
 
+# $1 - Helm charts dir
+define update-helm-schemas
+	@crd_schema=$$( $(YQ) eval -j '.spec.versions[0].schema.openAPIV3Schema | del(..|.description?)' $(CRD_PATH) ); \
+	crd_schema=$${crd_schema} $(YQ) eval -i -j 'env(crd_schema) * select(fileIndex == 0)' $(1)/scylla/values.schema.json; \
```

Comment:
why selecting the fileindex when there is only 1 file? is there a reason why we can't fully generate the file?

Reply:
We could easily generate the schema for `scylla` chart and I'm going to use this approach. As for the `scylla-manager` schema, I don't see a nice way to generate the entire file besides generating it from values, but that sort of defeats its purpose. I suppose we could have a `template.values.schema.json` file that we could merge with the generated `scylla` schema and output to `values.schema.json`. Wdyt?